ftpretty.py

```diff
@@ -31,18 +31,27 @@
 class ftpretty(object):
     """ A wrapper for FTP connections """
     conn = None
+    port = None
     tmp_output = None
     relative_paths = set(['.', '..'])
 
     def __init__(self, host, user, password,
             secure=False, passive=True, ftp_conn=None, **kwargs):
 
+        if 'port' in kwargs.keys():
```

Comment:
One last change, you don't need to use keys() here. You can simply check if 'port' in kwargs.